packages/loot-core/src/server/accounts/sync.js

```diff
@@ -351,6 +366,10 @@ async function normalizeNordigenTransactions(transactions, acctId) {
             trans.creditorAccount.iban.slice(-4) +
             ')',
         );
+        let transferPayee = resolvePayeeFromIban(trans.creditorAccount.iban);
```

Comment:
as I know some of the banks don't use `iban`, they have `bban` - doc: https://docs.google.com/spreadsheets/d/1ogpzydzotOltbssrc3IQ8rhBLlIZbQgm5QCiiNJrkyA/edit#gid=489769432
I think NordigenSandbox bank also doesn't use `iban` maybe we can try to match one of these two attributes?

Reply:
I wanted to do this and only decided against because I thought that no personal account would really have this but your spreadsheet proves me wrong :) IBAN has a standard format (2 letter country code + two check digits + bban) so perhaps it would make sense to extract BBAN from IBAN to allow lookups by BBAN when a full IBAN is known for an account and even vice versa. This could technically result in collisions but it's not very likely so maybe it would be worth for the added convenience.

Reply:
Just to give some data... Based on the spreadsheet, 1.5% of banks (15 out of 950) available in Nordigen don't provide `iban`. Out of those banks, only around half (8 out of 15) provides `bban`. Note that I have no data on whether `creditor/debtorAccount.bban` is provided in any cases where one makes a transfer from one account to another, either within the same bank or two different banks (which could make it useful to both try to get BBAN field and to try extracting BBAN from IBAN to allow lookups by BBAN useful).

I do find it hard to judge whether it is worth the hassle, especially considering my comment on the other PR (https://github.com/actualbudget/actual-server/pull/166#discussion_r1140959954) but I'm open to it if you think so.